Tests/Microsoft.AppCenter.Test.WindowsDesktop.Shared/Ingestion/Http/HttpNetworkAdapterTest.cs

```diff
@@ -1,48 +0,0 @@
-﻿using Microsoft.AppCenter.Ingestion.Http;
-using System.Net;
-using System.Diagnostics;
-using Xunit;
-
-namespace Microsoft.AppCenter.Test.WindowsDesktop.Ingestion.Http
-{
-    public class HttpNetworkAdapterTest
-    {
-        [Fact]
-        public void EnableTls12WhenDisabled()
-        {
-#if NET461
```

Comment:
Currently there is no 4.6 target in test projects. Also, it was invalid one because the changes applying in `static` initializer, not regular class constructor